.gitignore

```diff
@@ -3,3 +3,4 @@ composer.lock
 docs
 vendor
 coverage
+.phpunit.result.cache
```

Comment:
out of scope

Reply:
Tip: use a global .gitignore file ;)

Reply:
Have it already but I believe that phpunit is part of the project and because of that it should be in the project ignore!?